topostats/processing.py

```diff
@@ -274,9 +276,40 @@ def process_scan(
                                     **dnatracing_config,
                                 )
                                 tracing_stats[direction] = tracing_results["statistics"]
-                                # ordered_traces = tracing_results["ordered_traces"]
-                                # image_trace = tracing_results["image_trace"]
+                                ordered_traces = tracing_results["ordered_traces"]
+                                cropped_images = tracing_results["cropped_images"]
+                                image_trace = tracing_results["image_trace"]
                                 tracing_stats[direction]["threshold"] = direction
+
+                                # Plot traces for the whole image
+                                # Dilate the trace to be able to plot it without the trace being too thin
+                                dilation_iterations = int(filtered_image.images["gaussian_filtered"].shape[0] / 256)
```

Comment:
I think `dilation_iterations` would sit nicely as an option to the `Images()` class as that way users could choose the amount of dilation they wish to use, whether they are using the CLI or using `Image()` interactively in a Notebook.

Of course that entails adding more options to `plotting_dictionary.yaml` and validating via `validation.py` which is a but of a bind but worth it I think.